downstream/modules/platform/proc-configure-upgraded-aap.adoc

```diff
@@ -0,0 +1,14 @@
+[id="proc-configure-upgraded-aap_{context}"]
+
+= Configuring your upgraded {PlatformNameShort}
+
+== Configuring {ControllerName} instance groups
+
+After upgrading your {PlatformName} instance, associate your original instances to its corresponding instance groups by configuring settings in the {ControllerName} UI:
+
+. Log into the new Controller instance.
+. Content from old instance, such as credentials, jobs, inventories should now be visible on your Controller instance.
+. Navigate to *Administration* > *Instance Groups*.
```

Comment:
If this is a menu selection, the correct coding is:
Navigate to menu:Administration[Instance Groups]. 

This will render with the appropriate highlighting and > in the output.